README.md

```diff
@@ -6,3 +6,28 @@ This repository contains meta-documentation for the ORY Ecosystem. For individua
 - https://github.com/ory/hydra/tree/master/docs
 - https://github.com/ory/oathkeeper/tree/master/docs
 - https://github.com/ory/keto/tree/master/docs
+
+
+# Contributing Documentation
+
+For the ecosystem documents go to ory/docs.  
+For the individual projects docs go to ory/$project/docs   ($project = hydra,kratos,oathkeeper or keto)
+To test the docs locally:  
+```
+cd kratos/docs # for example for kratos
+npm i	# install dependencies
+npm start # test if everything works
+```
+To build the docs:
+
+`npm run build `
+
```

Comment:
Please format this a bit better. For example with `` ` `` . Also might be an idea to link the things.